src/app/plugins/connectfour.plugin.ts

```diff
@@ -0,0 +1,250 @@
+import { GuildMember, MessageEmbed, MessageReaction, ReactionCollector, User } from 'discord.js';
+import moment from 'moment';
+import { Plugin } from '../../common/plugin';
+import { ChannelType, IContainer, IMessage, Maybe } from '../../common/types';
+
+export class ConnectFourPlugin extends Plugin {
+  public name: string = 'Connect Four';
+  public description: string = 'Play connect four with a friend';
+  public usage: string = 'connectfour <playertag>';
+  public pluginAlias = ['connect4', 'connect', 'c4'];
+  public permission: ChannelType = ChannelType.Public;
+  public commandPattern: RegExp = /[^#]+#\d{4}/;
+
+  public static moves: string[] = ['1️⃣', '2️⃣', '3️⃣', '4️⃣', '5️⃣', '6️⃣', '7️⃣'];
+
+  constructor(public container: IContainer) {
+    super();
+  }
+
+  public async execute(message: IMessage, args: string[]) {
+    const guild = message.guild;
+    if (!guild) {
+      return;
+    }
+
+    const opponent = args.join(' ');
+    const oppMember = guild.members.cache.filter((m) => m.user.tag === opponent).first();
+    if (!oppMember) {
+      await message.reply('Could not find a user with that name.');
+      return;
+    }
```

Comment:
I think the user should get the other player by pinging them
That way its autocompleted and the other player is notified

so like `!c4 @Tanndlin`

Reply:
Just did it in [TTT](https://github.com/cs-discord-at-ucf/lion/blob/1a3a116132918d9ee019fba94d5272725dc35bea/src/app/plugins/tictactoe.plugin.ts#L14-L36)